jx_sqlite/query_table.py

```diff
@@ -278,7 +278,7 @@ def query(self, query):
 
             output = Data(
                 meta={"format": "table"},
-                header=column_names,
+                header=[split_field(h)[0] if "\\" in h else h for h in column_names],
```

Comment:
On removing check locally, split_field works on other types too where, hence creating a list out of  dot delimited column. I didn't see to it before but seems like this check is required after all.